.superpowers/brainstorm/83668-1775853195/content/component-preview.html

```
@@ -0,0 +1,110 @@
+<h2>New Components Preview</h2>
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)